src/main/java/org/hawkular/client/android/backend/BackendClient.java

```diff
@@ -290,6 +287,11 @@ public void getTriggers(@NonNull Callback<List<Trigger>> callback) {
         getPipe(pipeName).read(getFilter(uri), callback);
     }
 
+    @SuppressWarnings("unchecked")
+    private <T> void savePipe(String pipeName, Object object, Callback<List<T>> callback) {
```

Comment:
yes I am. I was shocked too. But Object was working good so just moved on.
Even retried after you said